### DISCUSSION TITLE
Server utility to get site URL

### DISCUSSION THREAD

**Discussion by kvnwolf** [sergiodxa/remix-utils#94] [Ideas]
Since many people should be trying to get the site URL in their apps, it would be a good idea to export it from `remix-utils`. I already have a working implementation and if this makes sense, I can send a PR :).

It could be exported as either `getURL` or `getRequestOrigin` to match with `document.location.origin`

**Comment by sergiodxa** [comment]
What’s the difference with this?
```ts
let { origin } = new URL(request.url)
```

**Reply by kvnwolf** [reply]
Duh! I was using [this other method](https://github.com/kentcdodds/kentcdodds.com/blob/main/app/utils/misc.tsx#L237-L245) to get it, but then I realized I also have the URL as part of the `Request` 😅
